### PR TITLE
Add DEV-GUIDE.md

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: DEV-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+- `<title>`
+  - summarize your change
+  - use imperative mood.
+  - Omit articles (`a/an` and `the`).
+  - should not exceed 75 characters
+  - must be capitalized
+- `<description>`
+  - detailed explantion of your change
```
Comment (TomTaehoonKim) [inline DEV-GUIDE.md:24]:
```suggestion
  - detailed explanation of your change
```

File: DEV-GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+We have to verify 3 things here:
+1. blah
+2. blah
+3. blah
+```
+
+## Pull Request Convention
+
+- For the title, use imperative mood and make it capitalized.
```
Comment (TomTaehoonKim) [inline DEV-GUIDE.md:39]:
```suggestion
- For the title, use imperative mood and capitalize it.
```

File: DEV-GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- Use **rebase only** (No merge or squash).
+- **NO MERGE COMMITS**.
+- Push your works on your own forked repo, and make a **PR across forks**.
+- Use the same title (head) and content (body) for a single-commit PR.
+- When the PR resolves some issues, put `fixes #123` in the content of the PR, not in the commit messages.
+- Conversation **must be resolved by the commentor**, not the PR author.
+- Do not take the Github notifications as the communication channel; use Discord.
```
Comment (TomTaehoonKim) [inline DEV-GUIDE.md:46]: how about adding spb-dev or spb-eng channel link?

Comment (junha1) [reply DEV-GUIDE.md:46]: This is for the main page. See #82

File: DEV-GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+<title>
+
+<description>
+```
+
+- `<title>`
+  - summarize your change
+  - use imperative mood.
+  - Omit articles (`a/an` and `the`).
+  - should not exceed 75 characters
```
Comment (lemonholic) [inline DEV-GUIDE.md:21]: I guess it's 50 characters?

**Review by lemonholic — APPROVED** [review]

**Review by TomTaehoonKim — APPROVED** [review]

**Pull request merged**